creates/order_change_status.js

```diff
@@ -39,7 +35,7 @@ module.exports = {
                 label:"New Status",
                 choices: {
                     "New": "New",
-                    "Hold for Confirm": "Hold for Confirm",
+                    "Hold for confirm": "Hold for confirm",
```

Comment:
Interesting that you can actually insert custom order statuses via the API ... we should probably handle that better in core.